airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/stream.py

```diff
@@ -564,26 +443,3 @@ def stream_slices(
             else:
                 # in case we have no files
                 yield None
-
-    def read_records(
```

Comment:
no need to override, now we can perform schema inference before incremental read since it wont take much time